articles/azure-functions/durable-functions-perf-and-scale.md

```diff
@@ -50,18 +50,18 @@ The following diagram illustrates how the Azure Functions host interacts with th
 
-As you can see, all VMs can compete for messages on the work item queue. However, only three VMs can acquire messages from control queues, and each VM locks a single control queue.
+As you can see, all VMs can compete for messages on the work-item queue. However, only three VMs can acquire messages from control queues, and each VM locks a single control queue.
 
 Orchestration instances are distributed across control queue instances by running an internal hash function against the orchestration's instance ID. Instance IDs are auto-generated and random by default that ensures that instances are balanced across all available control queues. The current default number of supported control queue partitions is **4**.
 
 > [!NOTE]
-> It's not currently possible to configure the number of partitions in Azure Functions. [Work to support this configuration option is being tracked](https://github.com/Azure/azure-functions-durable-extension/issues/73).
+> It's not currently possible to configure the number of control queue partitions in Azure Functions. [Work to support this configuration option is being tracked](https://github.com/Azure/azure-functions-durable-extension/issues/73).
 
 In general, orchestrator functions are intended to be lightweight and should not need a lot of computing power. For this reason, it is not necessary to create a large number of control queue partitions to get great throughput. Rather, most of the heavy work is done in stateless activity functions, which can be scaled out infinitely.
 
 ## Auto-scale
 
-As with all Azure Functions running in the Consumption plan, Durable Functions support auto-scale via the [Azure Functions scale-controller](https://docs.microsoft.com/azure/azure-functions/functions-scale#runtime-scaling). The Scale Controller monitors the length of the work-item queue and each of the control queues, adding or removing VM resources accordingly. If the control queue lengths are increasing over time, the scale controller will continue adding instances until it reaches the control queue partition count. If work item queue lengths are increasing over time, the scale controller will continue adding VM resources until it can match the load, regardless of the control queue partition count.
+As with all Azure Functions running in the Consumption plan, Durable Functions support auto-scale via the [Azure Functions scale controller](https://docs.microsoft.com/azure/azure-functions/functions-scale#runtime-scaling). The Scale Controller monitors the length of the work-item queue and each of the control queues, adding or removing VM instances accordingly. If the control queue lengths are increasing over time, the scale controller will continue adding VM instances until it reaches the control queue partition count. If work-item queue lengths are increasing over time, the scale controller will continue adding VM instances until it can match the load, regardless of the control queue partition count.
```

Comment:
VM "resources" is a bit vague and not consistent. Changed it to VM "instances" to be more specific and consistent.